### PR TITLE
FOUNDATIONS: Retrieve School By Id 

### PR DESCRIPTION
**Pull request by MahalakshmiMurari**

### CODE REVIEW

File: SchoolApp.ConsoleApp/Services/Foundations/Schools/SchoolService.Validations.cs [hunk 1]
```diff
@@ -56,7 +56,7 @@ private static void Validate(params (dynamic Rule, string Parameter)[] validatio
                         value: rule.Message);
                 }
             }
-
+
```
Comment (ShriHumrudha) [inline SchoolApp.ConsoleApp/Services/Foundations/Schools/SchoolService.Validations.cs:59]: run code cleanup

File: SchoolApp.ConsoleApp/Models/Schools/Exceptions/SchoolDependencyValidationException.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+// ------------------------------------------------
+
+using Xeptions;
+
+namespace SchoolApp.ConsoleApp.Models.Schools.Exceptions
+{
+    public class SchoolDependencyValidationException : Xeption
+    {
+        public SchoolDependencyValidationException(Xeption innerException)
+            : base(message: "School dependency validation error occured,fix the errors and try again.",
```
Comment (ShriHumrudha) [inline SchoolApp.ConsoleApp/Models/Schools/Exceptions/SchoolDependencyValidationException.cs:12]: check occurred spelling

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: SchoolApp.ConsoleApp/Models/Schools/Exceptions/SchoolDependencyValidationException.cs (hunk 2 above)
Comment (MahalakshmiMurari) [reply SchoolApp.ConsoleApp/Models/Schools/Exceptions/SchoolDependencyValidationException.cs]: Corrected spelling

File: SchoolApp.ConsoleApp/Services/Foundations/Schools/SchoolService.Validations.cs (hunk 1 above)
Comment (MahalakshmiMurari) [reply SchoolApp.ConsoleApp/Services/Foundations/Schools/SchoolService.Validations.cs]: ran code cleanup

**Review by ShriHumrudha — APPROVED** [review]

**Pull request merged**